setup.py

```diff
@@ -33,6 +33,8 @@
         'sphinx_rtd_theme',
         'setuptools_scm',
         'scipy>=1.0.0',
+        'crimson>=0.3.0',
```

Comment:
Please add `crimson>=0.3.0` to [requirements.txt](https://github.com/HumanCellAtlas/sctools/blob/master/requirements.txt).

This should also fix the failing Circle CI tests because they install requirements from the requirements.txt file: https://github.com/HumanCellAtlas/sctools/blob/master/.circleci/config.yml#L34